nodejs/webrtc-hello-video/public/webrtc_mgr.js

```diff
@@ -0,0 +1,521 @@
+const bandwidthRtc = new BandwidthRtc();
```

Comment:
I'm not super experienced with JavaScript so take my thoughts on this accordingly. One place to start as far as packaging would be to simply group related functions into separate files. If anything it would make them easier to work with and understand. Using "modern" Javascript we'd probably rely on exports and things like that so it wouldn't be much different in that regard (I don't think)

We can probably discuss this offline and in a group but those are my initial thoughts.

Reply:
yea, that was roughly my plan, but it's not super well thought out.